### PR TITLE
Add documentation customization

### PR DESCRIPTION
**Pull request by ullrichd21**
# Add documentation customization

## Description

Makes the documentation a little more `gator`. Refer to the before and after images below.

Before:
image

After:
image

## Type of Change

- [x] Feature
- [ ] Bug fix
- [x] Documentation

## Contributors

- @ullrichd21 

## Reminder

 - All GitHub Actions should be in a passing state before any pull request is merged.
 - All PRs must be reviewed by at least one team member and one member of the Integration team!
 - Any issues this PR closes are tagged in the description!

### CODE REVIEW

**Review by Michionlion — DISMISSED** [review]
I like these changes! I have two comments, however.

First, should `stylesheets/extra.css` be present in the repo? If not, how does `mkdocs` get this additional CSS? If it should be present in the repo, I'd suggest ensuring it's in `docs`, preferably `docs/gator.css` or something similar.

Second, is there a way for the bottom portion of the top green bar (the portion containing the "Home" and "Code Reference" links) to be a slightly different color, maybe a darker green? I find the whitespace to the far right of the bar (below the search and GitHub links) to be pretty ugly, and I think it could look better with a different color to de-emphasize the space.

Finally, thank you for this branding change! I think it will look great and make our documentation stand out just that little bit more! 🎉

**Review by laurennevill — COMMENTED** [review]
Wow, I love the efforts to gator-ify the documentation, it looks awesome! I do agree with Saejin that another shade of green could make this even better. Great work, @ullrichd21!

**Review by watson02 — COMMENTED** [review]
I also would have to agree with Saejin but other than that it looks great.

**Comment by ullrichd21** [comment]
So I made the changes requested.

image

About the missing CSS, just a mistake on my part! Thanks for pointing it out!

File: mkdocs.yml [hunk 1]
```diff
@@ -5,6 +5,18 @@ site_author: GatorGrade Team
...
+      primary: green
+  features:
+    - navigation.instant
+    - navigation.tabs
+    - toc.integrate
+    - navigation.top
+    - search.suggest
+    - search.highlight
+extra_css:
+  - stylesheets/gator.css
```
Comment (Michionlion) [inline mkdocs.yml:19]: And just to confirm, this is relative to the `docs` folder?

**Review by Michionlion — APPROVED** [review]
This looks good! Thanks for making those fixes!

**Review by burgess01 — APPROVED** [review]

**Pull request merged**